tree/ntuple/v7/inc/ROOT/RField.hxx

```diff
@@ -217,6 +217,10 @@ protected:
    /// are changed to their unencoded counterparts.
    void AutoAdjustColumnTypes(const RNTupleWriteOptions &options);
 
+   /// Factory method to resurrect a field from the stored on-disk type information.  This overload takes an already
+   /// normalized type name and type alias
```

Comment:
I guess that eventually we want to remove this method again and use a single ROOT function that resolves a type name. Let's add a TODO to remember.

Reply:
Hmmm, per [@pcanal's comment](https://github.com/root-project/root/pull/13255#issuecomment-1640517628), not sure about the future removal though.